src/Tests/dotnet-list-package.Tests/GivenDotnetListPackage.cs

```diff
@@ -21,6 +19,30 @@ public GivenDotnetListPackage(ITestOutputHelper output) : base(output)
         {
         }
 
+        [Fact]
+        public void ItShowsPackagesOnQuietVerbosity()
+        {
+            var testAssetName = "NewtonSoftDependentProject";
+            var testAsset = _testAssetsManager
+                .CopyTestAsset(testAssetName)
+                .WithSource();
+            var projectDirectory = testAsset.Path;
+
+            new RestoreCommand(testAsset)
+                .Execute()
+                .Should()
+                .Pass()
+                .And.NotHaveStdErr();
+
+            new ListPackageCommand(Log)
+                .WithWorkingDirectory(projectDirectory)
+                .Execute("--verbosity", "quiet")
+                .Should()
+                .Pass()
+                .And.NotHaveStdErr()
+                .And.NotHaveStdOutContaining("Newtonsoft.Json");
```

Comment:
The assertion in not too useful. If the input is diag, it will pass too. Either make it more specific. (that will "over reach" to nuget behavior) Or change to an parser level (there are some _parsertests_) tests to assert on the forwarded command.

Reply:
This should have been "contains" instead of "does not contain". essentially I was trying to test the integration between the .net CLI and nuget to ensure that the verbosity parameter is accepted. Really it would be enough just to verify that no error is returned. Just test fails right now on master branch because the nuget bits are too old.

Totally agree on the parser tests. I'll take a look.

Reply:
Fixed this test and added more detailed unit tests for the parser.

I think it's nice to have a test (this integration test here) that asserts an acceptable version of NuGet is being consumed. I can remove if you'd like though.